ID - Addressed Issues with 4DVD Logo and Time Series Menu

zaydor opened the pull request:
I tried something a bit different to try to get the 4DVD logo showing up. And also addressed formatting issues with the Close button from the last pull request.

dafrenchyman commented:
I'll get to this PR later in the week. Pretty busy right now.

dafrenchyman commented on src/app/timeseries-menu.component.html:
```diff
@@ -14,15 +14,26 @@ <h2 mat-dialog-title>Time Series</h2>
+    </button>
   </mat-dialog-content>
+  <h2 mat-dialog-title style="text-align: center">
+    <u>Pressure Layers</u>
```
This one addition (_Pressure Layers_ text) seems to be messing up the size of the pop-up window and adding arrow bars to it (to scroll up and down the window).

dafrenchyman commented on src/app/timeseries-menu.component.html:
```diff
@@ -14,15 +14,26 @@ <h2 mat-dialog-title>Time Series</h2>
+    <br />
+    <br />
+    <button mat-raised-button (click)="closeTimeSeries()">
+      Close Time Series
```
I don't like _Close Time Series_ button. I prefer how you put an X in the top right corner of the navbar to close that menu. Please do that here too, just add an X in the upper right corner of this window. I don't think we really need a close button, but that would look better.

dafrenchyman commented on src/app/timeseries-menu.component.html:
```diff
@@ -14,15 +14,26 @@ <h2 mat-dialog-title>Time Series</h2>
       Download Data
     </button>
+    <br />
+    <p>Latitude: {{ this._model.settings.CurrGridBoxLat }}</p>
```
I like adding the Lat/Lon to this window. Nice job.

dafrenchyman commented on src/app/settings.ts:
```diff
@@ -193,6 +193,18 @@ export class Settings {
     return null;
   }
 
+  StringToArray(str): string {
```
I like the fact that you did this function. But the function/methods name should match what it's doing. It's not doing a `StringToArray`, it's actually generating the title.

zaydor commented:
Addressed your comments, also moved the pressure layer checkboxes to the right of the graph as I thought it looked a bit cleaner than below the graph.

dafrenchyman commented:
Opps,.. just noticed this. I'll get to it. Sorry.

zaydor commented:
Yeah, apologies. I am aware of that issue. It was working before without extending the popup box, but for whatever reason it turned into a single list. Working on a fix for it but was not able to test it while out of town. Will be fixed by Monday.

dafrenchyman commented:
Hi Zaydor, 
I'm finally getting around to testing this, and the time series screen still doesn't work right.
I think we need to go about this a completely different way:
- You should split your work into different PRs. The other stuff you've done: The logo, the changes to removing `mrsharky_` would all have been easily approved if they were in their own PRs. When you write PRs they should be for changing/adding one thing, especially in this case when those would have been easy to approve.
- I do like what you've done with the time series, the only problem is that the window re-renders when it adds the legend in. I don't see the scroll lines anymore (which is good), but depending on the window size I have, I don't see all the buttons available. I think you have two options here: 
-- Remove the legend, and make our buttons the legend instead. Color code our legend buttons to match what they look like in the graph.
-- Keep the legend buttons on the bottom for now, but keep your lat/lon stuff. Eventually I think it would be beneficial to move to a different charting library (https://c3js.org/)
